Makefile

```
@@ -318,7 +318,7 @@ proto-update-deps:
 ###############################################################################
 
 build-docker-local-simapp:
-	@$(MAKE) -C networks/local
+	@docker build -t cosmos-sdk/simapp .
```

```suggestion
	docker build -t cosmos-sdk/simapp .
```

IMO it's always useful to see what commands are launched by a makefile's target

---

agreed will make the change